internal/controllers/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -428,16 +449,36 @@ func (r *Reconciler) patchUnhealthyTargets(ctx context.Context, logger logr.Logg
 				// Create the external clone.
 				if err := r.Client.Create(ctx, to); err != nil {
 					conditions.MarkFalse(m, clusterv1.ExternalRemediationRequestAvailableCondition, clusterv1.ExternalRemediationRequestCreationFailedReason, clusterv1.ConditionSeverityError, err.Error())
+
+					v1beta2conditions.Set(t.Machine, metav1.Condition{
+						Type:   clusterv1.MachineExternallyRemediatedV1Beta2Condition,
+						Status: metav1.ConditionFalse,
+						Reason: clusterv1.MachineOwnerRemediateExternalRemediationRequestCreationFailedV1Beta2Reason,
```

Comment:
Do we want to add a message? E.g. like in Machine conditions we have that standard message about ~ "take a look at  the controller logs"